Added migration docs (#1442)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/migrate-to-okta/bulk-migration-with-credentials/index.md

* [User status values](https://developer.okta.com/docs/reference/api/users/#user-status) (API reference)
* [Activate User request](https://developer.okta.com/docs/reference/api/users/#activate-user) (API reference)
* [End user account states](https://help.okta.com/en/prod/Content/Topics/Directory/end-user-states.htm) (product documentation)
* [Activate user accounts](https://help.okta.com/en/prod/Content/Topics/Directory/ad-agent-activate-user-accounts.htm) (product documentation)

### Rate limits

Remember that [rate limits](https://developer.okta.com/docs/reference/rate-limits/) apply to API requests when doing bulk/batch user migration, and the rate limits differ depending on the level of service you have purchased from Okta. [You can check your rate limits](https://developer.okta.com/docs/reference/rate-limits/#check-your-rate-limits-with-okta-s-rate-limit-headers) in your code using Okta’s Rate Limit Headers.

###Next steps

At this point, you should understand how to use the Okta API to migrate legacy users and groups to Okta.

> Your next step should be configuring the necessary integration and access to applications for your users. Be sure to read the product documentation for an [overview of application integration](https://help.okta.com/en/prod/Content/Topics/Apps/Apps_Apps.htm) and see the information about [The Applications Page](https://help.okta.com/en/prod/Content/Topics/Apps/Apps_Apps_Page.htm) for more on how to configure your applications.

Reviewer: ```suggestion
Your next step should be configuring the necessary integration and access to applications for your users. Be sure to read the product documentation for an [overview of application integration](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_Apps) and see the information about [The Applications Page](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_Apps_Page) for more on how to configure your applications.
```

### Reference 

This is a collection of reference links that we covered in this guide, as well as links for further information:

#### API reference pages

> * [Okta Users API](https://developer.okta.com/docs/reference/api/users/)

Reviewer: ```suggestion
* [Okta Users API](/docs/reference/api/users/)
```

* [Okta Groups API](https://developer.okta.com/docs/reference/api/groups/)

> * [API error codes](https://developer.okta.com/docs/reference/error-codes/)

Reviewer: ```suggestion
* [API error codes](https://developer.okta.com/docs/reference/error-codes/)
```
```suggestion
* [API error codes](/docs/reference/error-codes/)
```

> * [Default user profile properties](https://developer.okta.com/docs/reference/api/users/#default-profile-properties)

Reviewer: ```suggestion
* [Default user profile properties](/docs/reference/api/users/#default-profile-properties)
```

> * [Rate limits](https://developer.okta.com/docs/reference/rate-limits/)

Reviewer: ```suggestion
* [Rate limits](/docs/reference/rate-limits/)
```

#### Product help pages

> * [Get started with Okta](https://help.okta.com/en/prod/Content/Topics/Security/admin-get-started.htm)

Reviewer: ```suggestion
* [Get started with Okta](https://help.okta.com/en/prod/okta_help_CSH.htm#ext-get-started)
```

That alias should work by the time this PR is merged.

> * [Profile mastering](https://help.okta.com/en/prod/Content/Topics/Directory/eu-profile-masters.htm)

Reviewer: ```suggestion
* [Profile mastering](https://help.okta.com/en/prod/okta_help_CSH.htm#csh-profile-masters)
```

> * [Users, Groups, and Profiles](https://help.okta.com/en/prod/Content/Topics/Directory/eug-overview.htm)

Reviewer: ```suggestion
* [Users, Groups, and Profiles](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_User_Lifecycle_Overview)
```
